docs/volunteering/glossary.md

```diff
@@ -0,0 +1,66 @@
+# Glossary
+
+While we try to use terminology which are clear and/or well known, here's a list
+you can check if you're not sure!
+
+<!---
+Note to editors; please:
+- keep this list in alphabetical order
+- ensure that, within the list, usages of other terms defined in the list are
+  emphasised suitably
+--->
+
+_Competition programme_:
+
+: the overall programme of events and activities which the teams participate in
+
+_Competition year_:
+
+: the 6 months (or so) between _Kickstart_ and the _Competition_ during which
+teams build their robots; see also _SRnnnn_
+
+_Core Team_:
+
+: the group of 8 or so volunteers responsible for delivering a particular year's
+_Competition programme_.
+
+[_GitHub_](https://github.com/srobo):
+
+: a platform for hosting code repositories which we also use to coordinate tasks
+
+_Kickstart_:
+
+: the start of the competition year, when we hand out kits to teams
+
+_Mentor_:
+
+: a volunteer who helps teams build their robots
+
+_Shepherd_:
+
+: a volunteer at the competition event who helps ensure teams make it to their
+matches on time
+
+[_Slack_][https://studentrobotics.slack.com/]:
+
+: an instant-messaging tool we use for quick discussions
+
+_SR_:
+
+: a shortened form of Student Robotics
+
+_srobo_:
+
+: another shortened form of Student Robotics
+
+_SRnnnn_ (e.g: SR2019):
+
+: this is the canonical way to refer to a given year's competition programme;
+the year number is the calendar year in which the competition event takes place,
+so SR2019 was the _competition programme_ which ran from November 2018 until
+April 2019
+
+_Tech Day_
+
+: during the _competition year_ we organise events at which the teams can come
+with their robots and get extra help from _mentors_ as well as meet other teams
```

Comment:
Some glossary suggestions:

- TLA
- SRComp
- Rookie Team
- Token, and how this differentiates from a Marker

Reply:
Hrm.
I'm not sure there's a benefit in this glossary merely encompassing all possible terms (i.e: a dictionary), nor do I want it to become either a duplicate of the rulebook or other resources.

For example: I'd hope that "SRComp" would be sufficiently googleable and "Rookie Team" clear enough from its constituent words' meaning.

Will have a think about how to draw the line here.

Reply:
Turns out: "SRComp" is semi-googleable. You do get some results which I'd call "right", but many which aren't.

Reply:
4b3e20a